OfficeDevPnP.Core/OfficeDevPnP.Core/Framework/Provisioning/Model/HashFormatters/HashFormatter.cs

```diff
@@ -0,0 +1,44 @@
+﻿using OfficeDevPnP.Core.Framework.Provisioning.Model.Attributes;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OfficeDevPnP.Core.Framework.Provisioning.Model.HashFormatters
+{
+    public static class HashFormatter<T>
+    {
+        public static int GetFormatter(T model)
```

Comment:
The reflection below is probably not super efficient. Anyone know a better way to do it?